ims-react/.eslintrc.js

```diff
@@ -0,0 +1,36 @@
+module.exports = {
+    "env": {
+        "browser": true,
+        "es6": true
+    },
+    "extends": [
+        "eslint:recommended",
+        "plugin:react/recommended"
+    ],
+    "globals": {
+        "Atomics": "readonly",
+        "SharedArrayBuffer": "readonly"
+    },
+    "parser": "babel-eslint",
```

Comment:
I had to switch the parser out.  ESLint was not compatible with the most up-to-date node/es environemnt.